Makefile

```diff
@@ -6,6 +6,17 @@ help: ## Display this help text
 
 lint: black flake8 isort mypy ## Lint the project
 
+test:
+	tests/scripts/docker/run_tests.sh
+
+coverage: PYTEST_ARGS=--cov --cov-context=test --cov-config=.coveragerc --cov-branch
+coverage: export COVERAGE_FILE=.coverage
+coverage: test
+
+coverage-ci: PYTEST_ARGS=--cov --cov-context=test --cov-config=.coveragerc --cov-branch
+coverage-ci: export COVERAGE_FILE=.coverage
+coverage-ci: test-ci
```

Comment:
`test-ci` is missing, therefore `coverage-ci` is duplicated, is that intentional?